src/Tizen.Sensor/Tizen.Sensor/SensorEnumerations.cs

```diff
@@ -354,4 +362,57 @@ public enum AutoRotationState
         /// <since_tizen> 7 </since_tizen>
         Degree_270 = 4
     }
+
+
+    /// <summary>
+    /// HeartRateMonitorBatchState types.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public enum HeartRateMonitorBatchState
+    {
+        /// <summary>
+        /// Flush but there was no batched data
```

Comment:
Do you have any reason why you use past tense?
I think the present tense is good because the enum indicates a certain state.
If you agree with me, please change the other descriptions.